docs/getting-started/using-gnokey.md

```diff
@@ -0,0 +1,777 @@
+GnoWeb      ┃ I gnoweb started lisn=http://127.0.0.1:8888
+-- READY   ┃ I for commands and help, press `h`
+```
+
+Now we have a local Gno node listening on `127.0.0.1:26657` with chain ID `dev`,
+which we can use to upload our code to.
+
+Next, let's configure the `addpkg` subcommand. Assuming we are in the `example/p`
+folder, the command will look like this:
+
+```bash
+gnokey maketx addpkg \                                                                                                                                                                                          
+-pkgpath "gno.land/p/<your_namespace>/hello_world" \
+-pkgdir "." \
+-send "" \
+-gas-fee 10000000ugnot \
+-gas-wanted 8000000 \
+-broadcast \
+-chainid dev \
+-remote "127.0.0.1:26657" \
+```
+
+Once we have added a desired namespace to upload the package to, we can specify
+a keypair name to use to execute the transaction:
+
+```bash
+gnokey maketx addpkg \                                                                                                                                                                                          
+-pkgpath "gno.land/p/leon/hello_world" \
+-pkgdir "." \
+-send "" \
+-gas-fee 10000000ugnot \
+-gas-wanted 200000 \
+-broadcast \
+-chainid dev \
+-remote "127.0.0.1:26657" \
+mykey
+```
+
+If the transaction was successful, you will get the following output from `gnokey`:
+
+```
+OK!
+GAS WANTED: 200000
+GAS USED:   117564
+HEIGHT:     3990
+EVENTS:     []
+```
+
+Let's analyze the output, which is standard for any `gnokey` transaction:
+- `GAS WANTED: 200000` - the original amount of gas specified for the transaction
+- `GAS USED:   117564` - the gas used to execute the transaction
+- `HEIGHT:     3990` - the block number at which the transaction was executed at
+- `EVENTS:     []` - events emitted by the transaction, in this case, none
+
+Congratulations! You have just uploaded a pure package to your local chain.
+If you wish to upload the package to a remote testnet, make sure to switch out
+the `-chainid` & `-remote` values for the ones matching your desired testnet.
+Find a list of all networks in the [Network Configuration](../reference/network-config.md)
+section.
+
+### `Call`
+
+The `Call` message type is used to call any exported function on the chain.
+You can send a `Call` transaction with `gnokey` using the following command:
+
+```bash
+gnokey maketx call
+```
+
+:::info `Call` uses gas
+
+Using `Call` to call an exported function will use up gas, even if the function
+does not modify on-chain state. If you are calling such a function, you can use
+the [`query` functionality](#query) for a read-only call which does not use gas.
+
+:::
+
+For this example, we will call the `wugnot` realm, which wraps GNOTs to a
+GRC20-compatible token called `wugnot`. We can find this realm deployed on the
+[Portal Loop](../concepts/portal-loop.md) testnet, under the `gno.land/r/demo/wugnot`.
```

Comment:
What do you think about writing and deploying a single Realm that you can use to demonstrate all of the functionalities in this tutorial?

Reply:
I think it's nice to use the tutorial to also show the reader some of the realms that they can actually use on the blockchain, so I would like to keep it this way